### DISCUSSION TITLE
Editor and author role permissions

### DISCUSSION THREAD

**Discussion by alexpi** [strapi/strapi#7364] [Legacy Topics]
Hello,

I need to give my client the ability to edit content, so I created an editor user.
In settings > roles, I see that Editors and Authors have the exact same capabilities, and the ability to create/update collection content is not included. The Collection Types CRUD settings are greyed out. Only the Single Types are checked.

Is this the normal behavior? Logging in as editor, one can only upload media.

**Comment by derrickmehaffy** [comment]
Hello,

This is intended to allow users to select which roles are allowed to modify the models, you can select which model and fields you want to allow the role to modify, but the CE version doesn't give granular control to what type of action the role can make on a field. There is some policies pre-applied to each role for field actions however.
